### PR TITLE
Flowsheets documentation ADM1, ASM1, ASM2d

### PR DESCRIPTION
**Pull request by agarciadiego**
## Fixes/Resolves:

#1219

## Summary/Motivation:

- Adds documentation for ASM1
- Adds documentation for ASM2d
- Adds documentation for ADM1

## Changes proposed in this PR:
-
-

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: docs/technical_reference/flowsheets/ADM1.rst [hunk 1]
```diff
@@ -0,0 +1,102 @@
+Anaerobic Digestion Model No. 1
+===============================
+
+Introduction
+------------
+
+The IWA created Anaerobic Digestion Model No. 1 with a mathematical model to predict anaerobic digestion scenarios.
```
Comment (MarcusHolly) [inline docs/technical_reference/flowsheets/ADM1.rst]:
```suggestion
The IWA created Anaerobic Digestion Model No. 1 (ADM1) with a mathematical model to predict anaerobic digestion scenarios. 
```

Comment (MarcusHolly) [reply docs/technical_reference/flowsheets/ADM1.rst]:
```suggestion
The International Water Association (IWA) created Anaerobic Digestion Model No. 1 with a mathematical model to predict anaerobic digestion scenarios. 
```

File: docs/technical_reference/flowsheets/ASM1.rst [hunk 2]
```diff
@@ -0,0 +1,124 @@
+Activated Sludge Model No. 1
+============================
+
+Introduction
+------------
+
+Activated Sludge Model No. 1 was created by the IWA to create a mathematical model to predict the output of activated sludge 
+models. The model aims to incorporate carbon oxidation, nitrification, and denitrification. The model quantifies 
+the kinetics and stoichiometry of each process. The property and reaction relationships `ASM1
+<https://watertap.readthedocs.io/en/latest/technical_reference/property_models/ASM1.html>`_
```
Comment (MarcusHolly) [inline docs/technical_reference/flowsheets/ASM1.rst]:
```suggestion
Activated Sludge Model No. 1 (ASM1) was created by the International Water Association (IWA) to create a mathematical model to predict the output of activated sludge 
models. The model aims to incorporate carbon oxidation, nitrification, and denitrification. The model quantifies 
the kinetics and stoichiometry of each process. The `ASM1
<https://watertap.readthedocs.io/en/latest/technical_reference/property_models/ASM1.html>`_ property and reaction relationships 
```

File: docs/technical_reference/flowsheets/ASM1.rst [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+are based on the relationships provided for the activated sludge biological reactor provided in `Henze, M. et al. (1987) 
+<https://belinra.inrae.fr/doc_num.php?explnum_id=4467>`_.
```
Comment (MarcusHolly) [inline docs/technical_reference/flowsheets/ASM1.rst]:
```suggestion
are based on the relationships provided for the activated sludge biological reactor provided in `Henze, M. et al. (1987) 
<https://belinra.inrae.fr/doc_num.php?explnum_id=4467>`_.
```

Can remove one of the "provided" from this sentence

File: docs/technical_reference/flowsheets/ASM2d.rst [hunk 4]
```diff
@@ -0,0 +1,136 @@
+Activated Sludge Model No. 2d
+=============================
+
+Introduction
+------------
+
+The Activated Sludge Model No. 2d represents a variation of the ASM1 model to account for biological phosphorus  with 
+nitrification-denitrification. The property and reaction relationships `ASM2d
+<https://watertap.readthedocs.io/en/stable/technical_reference/property_models/ASM2D.html>`_
```
Comment (MarcusHolly) [inline docs/technical_reference/flowsheets/ASM2d.rst]:
```suggestion
The Activated Sludge Model No. 2d (ASM2d) represents a variation of the ASM1 model to account for biological phosphorus  with 
nitrification-denitrification. The  `ASM2d
<https://watertap.readthedocs.io/en/stable/technical_reference/property_models/ASM2D.html>`_ property and reaction relationships 
```

File: docs/technical_reference/flowsheets/ASM2d.rst [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+    * `CSTR <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/cstr.html>`_
+    * Aeration tank
+    * Secondary clarifier
+
+Documentation for the property model can be found below.
+    * `ASM2d <https://watertap.readthedocs.io/en/stable/technical_reference/property_models/ASM2D.html>`_
+
+Degrees of Freedom
+------------------
+The following variables are initially specified for simulating the ASM1 flowsheet (i.e., degrees of freedom = 0):
```
Comment (MarcusHolly) [inline docs/technical_reference/flowsheets/ASM2d.rst]:
```suggestion
The following variables are initially specified for simulating the ASM2d flowsheet (i.e., degrees of freedom = 0):
```

File: docs/technical_reference/flowsheets/ASM2d.rst [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+   "Secondary clarifier X_PP split fraction", "0.02144", ":math:`\text{dimensionless}`"
+   "Secondary clarifier X_S split fraction", "0.02221", ":math:`\text{dimensionless}`"
+   "Secondary clarifier X_TSS split fraction", "0.02194", ":math:`\text{dimensionless}`"
+   "Separator recycle split fraction", "0.97955", ":math:`\text{dimensionless}`"
+   "Recycle pump outlet pressure", "101325", ":math:`\text{Pa}`"
+
+Future Refinements
+------------------
+
+The following modifications to ASM1 are planned for development:
```
Comment (MarcusHolly) [inline docs/technical_reference/flowsheets/ASM2d.rst]:
```suggestion
The following modifications to ASM2d are planned for development:
```

File: docs/technical_reference/flowsheets/ASM2d.rst [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+    * Improving costing relationships in terms of detail, completeness, and reasonable validity
+    * Accounting for temperature-dependence in the oxygen mass transfer coefficient (KLa) and oxygen concentration at saturation
+    * Replacing the ideal-separator formulation in the secondary clarifier with the widely used double-exponential settling model (i.e., the Takacs model)
+    
+**NOTE: The repository also contains flowsheets for ASM2d modified for phosphorous removal**
```
Comment (MarcusHolly) [inline docs/technical_reference/flowsheets/ASM2d.rst]: Include a link here that points to the modified ASM2d documentation?

File: docs/technical_reference/flowsheets/ADM1.rst [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+The IWA created Anaerobic Digestion Model No. 1 with a mathematical model to predict anaerobic digestion scenarios. 
+The objective of the mathematical model is to simulate biological reactions simultaneously with the disintegration, uptake, and decay of microorganisms
+in anaerobic sludge. The property and reaction relationships `ADM1
+<https://watertap.readthedocs.io/en/latest/technical_reference/property_models/ADM1.html>`_
+of an anaerobic digestion model for industrial wastewater treatment from food and beverage production using an anaerobic digester as provided in 
+`Batstone, D. J. et al. (2002) 
+<https://iwaponline.com/wst/article-abstract/45/10/65/6034/The-IWA-Anaerobic-Digestion-Model-No-1-ADM1?redirectedFrom=fulltext>`_ and `Rosen, C. et al. 
+(2006) <https://pubmed.ncbi.nlm.nih.gov/17037165/>`_
+Implementation
+--------------
```
Comment (MarcusHolly) [inline docs/technical_reference/flowsheets/ADM1.rst:17]:
```suggestion

Implementation
--------------
```

I believe there needs to be a blank line before section headings for it to display correctly.

File: docs/technical_reference/flowsheets/ADM1.rst [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+biogas stream and residual sludge. S_h2 and S_ch4 exist in both vapor phase and liquid phase, S_co2 exists vapor phase, and the other components 
+only exist in the liquid phase. The amount of CO2 dissolved in the liquid phase is equivalent to S_IC - S_HCO3-.
+The flowsheet relies on the following key assumptions:
+
+   * supports steady-state only
+   * property and reaction packages are provided for the anaerobic digester model (ADM)
+
+.. figure:: ../../_static/flowsheets/ADM1.png
+    :width: 800
+    :align: center
```
Comment (MarcusHolly) [inline docs/technical_reference/flowsheets/ADM1.rst]: This image looks really big in the documentation... is there an easy way to shrink it a little bit?

**Review by MarcusHolly — COMMENTED** [review]
In general things look good, just a few minor things to address.

File: docs/technical_reference/flowsheets/ADM1.rst [hunk 10]
```diff
@@ -0,0 +1,103 @@
...
+of an anaerobic digestion model for industrial wastewater treatment from food and beverage production using an anaerobic digester as provided in 
+`Batstone, D. J. et al. (2002) 
+<https://iwaponline.com/wst/article-abstract/45/10/65/6034/The-IWA-Anaerobic-Digestion-Model-No-1-ADM1?redirectedFrom=fulltext>`_ and `Rosen, C. et al. 
+(2006) <https://pubmed.ncbi.nlm.nih.gov/17037165/>`_
+
+Implementation
+--------------
+
+Figure 1 shows the process flow diagram for ADM1 where the flowsheet only includes the anaerobic digester ehich processes sludge to produce a 
+biogas stream and residual sludge. S_h2 and S_ch4 exist in both vapor phase and liquid phase, S_co2 exists vapor phase, and the other components
```
Comment (MarcusHolly) [inline docs/technical_reference/flowsheets/ADM1.rst]:
```suggestion
Figure 1 shows the process flow diagram for ADM1 where the flowsheet only includes the anaerobic digester which processes sludge to produce a 
biogas stream and residual sludge. S_h2 and S_ch4 exist in both vapor phase and liquid phase, S_co2 exists in the vapor phase, and the other components 
```

**Review by MarcusHolly — APPROVED** [review]
LGTM - a few more minor typos to correct

**Review by luohezhiming — APPROVED** [review]
LGTM

**Pull request merged**